src/OsmMom.sol

```diff
@@ -75,23 +71,39 @@ contract OsmMom {
 
     mapping (bytes32 => address) public osms;
 
-    constructor() public {
+    constructor(address vat_) {
+        vat = vat_;
         owner = msg.sender;
+        emit SetOwner(msg.sender);
+    }
+
+    function file(bytes32 what, address data) external onlyOwner {
+        if (what == "autoLine") {
+            autoLine = data;
+        } else {
+            revert("OsmMom/file-unrecognized-param");
+        }
     }
 
-    function setOsm(bytes32 ilk, address osm) external note onlyOwner {
+    function setOsm(bytes32 ilk, address osm) external onlyOwner {
         osms[ilk] = osm;
+        emit SetOsm(ilk, osm);
     }
 
-    function setOwner(address owner_) external note onlyOwner {
+    function setOwner(address owner_) external onlyOwner {
```

Comment:
Consider distinguishing the permissions changing functions `setOwner` and `setAuthority` by moving them to after isAuthorized (or at least right after the constructor). I know this has been like this before but there are now more things to set (after adding the AutoLine). Also if you do that we might want to reorder the events declarations as well.

Reply:
I moved them right below the constructor. To keep all the admin (owner) functions together but give them the primary relevance (as we use to do with `rely`/`deny`).
I've reordered the events as well.